process_report/tests/unit/test_chromium.py

```diff
@@ -0,0 +1,63 @@
+import os
+import shutil
+import tempfile
+from unittest import TestCase
+
+import pandas as pd
+
+from process_report.invoices.pi_specific_invoice import PIInvoice
+from process_report.invoices import invoice
+
+
+class TestPIInvoiceExport(TestCase):
+    def setUp(self):
+        self.output_dir = tempfile.mkdtemp()
+        self.invoice_month = "2024-01"
+
+        self.df = pd.DataFrame(
+            [
+                {
+                    invoice.INVOICE_DATE_FIELD: "2024-01-01",
+                    invoice.PROJECT_FIELD: "ProjectX",
+                    invoice.PROJECT_ID_FIELD: "PA-001",
+                    invoice.PI_FIELD: "Jane Doe",
+                    invoice.INVOICE_EMAIL_FIELD: "jane.doe@university.edu",
+                    invoice.INVOICE_ADDRESS_FIELD: "123 Campus Dr, Cityville",
+                    invoice.INSTITUTION_FIELD: "Test University",
+                    invoice.INSTITUTION_ID_FIELD: "TU-001",
+                    invoice.SU_HOURS_FIELD: 50,
+                    invoice.SU_TYPE_FIELD: "GBhr",
+                    invoice.RATE_FIELD: 2.0,
+                    invoice.GROUP_NAME_FIELD: "PrepaidGroupX",
+                    invoice.GROUP_INSTITUTION_FIELD: "Test University",
+                    invoice.GROUP_BALANCE_FIELD: 1000,
+                    invoice.COST_FIELD: 100,
+                    invoice.GROUP_BALANCE_USED_FIELD: 200,
+                    invoice.CREDIT_FIELD: 20,
+                    invoice.CREDIT_CODE_FIELD: "CRED123",
+                    invoice.BALANCE_FIELD: 80,
+                    invoice.IS_BILLABLE_FIELD: True,
+                    invoice.MISSING_PI_FIELD: False,
+                }
+            ]
+        )
+
+    def tearDown(self):
+        shutil.rmtree(self.output_dir)
+
+    def test_pi_invoice_pdf_generation(self):
+        pi_invoice = PIInvoice(
+            name=self.output_dir, invoice_month=self.invoice_month, data=self.df
+        )
+        pi_invoice.process()
+        pi_invoice.export()
+
+        pdf_files = [f for f in os.listdir(self.output_dir) if f.endswith(".pdf")]
+        self.assertGreater(len(pdf_files), 0)
```

Comment:
From the test dataframe you've made, only one PDF should be printed. Assert that there's only exactly one PDF. Also, please check that it contains the expected filename